GitUI/CommandsDialogs/SettingsDialog/CheckSettingsLogic.cs

```diff
@@ -177,6 +177,12 @@ bool TestGitCommand(string command)
             {
                 try
                 {
+                    // Use cached version if possible
+                    if (AppSettings.GitCommand == command && !(GitVersion.Current?.IsUnknown ?? true))
```

Comment:
Does this
```suggestion
                    if (AppSettings.GitCommand == command && GitVersion.Current?.IsUnknown is false)
```
work, too?

Reply:
works fine, updating